CHANGELOG.md

```diff
@@ -64,6 +66,8 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 ### Deprecated
 - **Sequence**:
   - Marked `map(by:)`, `compactMap(by:)`, `filter(by:)` as deprecated in favor use of Key Path expressions as functions feature in Swift 5.2. [#862](https://github.com/SwifterSwift/SwifterSwift/pull/862) by [Roman Podymov](https://github.com/RomanPodymov).
+- **UIButton**
+  - Marked `centerTextAndImage(imageAboveText:spacing:)` as deprecated. [#903](https://github.com/SwifterSwift/SwifterSwift/pull/903) by [Jayxiang](https://github.com/Jayxiang).
```

Comment:
Can you extend this by "as deprecated in favor of new added methods `setImageLeftText(spacing:)`, `setImageRightText(spacing:)`, `setImageAboveText(spacing:)`, `setImageBelowText(spacing:)`"?

Reply:
ok